packages/codemirror/src/editor.ts

```diff
@@ -25,6 +25,7 @@ import {
   IEditorLanguageRegistry,
   IExtensionsHandler
 } from './token';
+import { dropCursor } from '@codemirror/view';
```

Comment:
Maybe this import can be grouped with the existing `@codemirror/view` import above?